politeiad/plugins/comments/comments.go

```diff
@@ -55,6 +59,13 @@ const (
 	// SettingAllowExtraData is the default value of the bool flag which
 	// determines whether posting extra data along with the comment is allowed.
 	SettingAllowExtraData = false
+
+	// SettingVotesPageSize is the default maximum number of comment votes
+	// that can be returned at any one time. It defaults to 2621 to limit the
+	// comment votes route payload size to be ~1MiB, as each comment vote size
+	// is expected to be around 400 bytes which means:
+	// 2621 * 400 byte = ~1048576 byte = 1MiB.
+	SettingVotesPageSize uint32 = 2621
```

Comment:
Good docs.  Let's make the page size a round number though (2500).  If you're a consumer of this API, 2621 is oddly specific.  Round numbers are just nicer for page sizes.